src/utils/signatureIsValid.ts

```diff
@@ -0,0 +1,22 @@
+import { createHmac } from 'crypto';
+import { getJiterConfig } from '../config';
+
+type SignatureIsValidArgs = {
+  signature: string;
+  body: string | Record<any, any>;
```

Comment:
```suggestion
  payload: string;
  ```

we are signing the payload only